mantidimaging/gui/windows/spectrum_viewer/presenter.py

```diff
@@ -232,6 +238,7 @@ def redraw_all_rois(self) -> None:
             widget_roi = self.view.spectrum_widget.get_roi(roi_name)
             spectrum = self.model.get_spectrum(widget_roi, self.spectrum_mode, self.view.shuttercount_norm_enabled())
             self.view.set_spectrum(roi_name, spectrum)
+        self.update_displayed_image(autoLevels=False)
```

Comment:
Does this need calling here, it does not look like the old code did an image redraw here.